<?xml version="1.0"?>
<launch>
  <arg name="source_frame" default="/photoneo_center" />
  <arg name="target_frame" default="/base_link" />
  <node pkg="oneshot_calibration_system" type="icp_calibrator" name="icp_calibrator" output="screen" >
    <remap from="input_data" to="icp_data_array" />
    <param name="target_frame" value="$(arg target_frame)" />
    <param name="source_frame" value="$(arg source_frame)" />
  </node>
</launch>
